settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'dfbnc'
```

Comment:
Don't really need this - Gradle will name the root project after the folder its in, which should be fine for DFBnc.

We have it in DMDirc because our repository is stupidly called 'meta'...